src/outputs/short_form_out.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import numpy as np
+
+
+def create_new_cols(df: pd.DataFrame) -> pd.DataFrame:
+    """Created blank columns & year column for short form output
+
+    Args:
+        df (pd.DataFrame): short form output data frame
+
+    Returns:
+        pd.DataFrame: returns short form output data frame with added new columns
+    """
+    columns_names = [
+        "freeze_id",
+        "inquiry_id",
+        "period_contributor_id",
+        "post_code",
+        "ua_county",
+        "wowentref",
```

Comment:
"wowentref" - "wowenterprisereference" seems to already be in the dataset so this might result in duplicate columns. 
I'm unclear on the meaning behind the table detailed on the ticket.